\documentclass[16pt]{article}
\usepackage{blindtext}
\usepackage{hyperref}
\usepackage{background}
\usepackage{fancyhdr}
\usepackage{graphicx}
\graphicspath{{images/}}

\backgroundsetup{contents=Surge Report,opacity=0.30,scale=4,color=blue,angle=40}

\title{Surge MidTerm Report}
\usepackage{hyperref}
\author{Milan Anand Raj\\manandraj20@iitk.ac.in}

\date{June 24, 2022}

\pagestyle{fancy}
\fancyhf{}
\fancyhead[LE,RO]{Surge Report}
\fancyhead[RE,LO]{MidTerm}
\fancyfoot[CE,CO]{\leftmark}
\fancyfoot[LE,RO]{\thepage}

\begin{document}
%\SetWatermarkText{Surge Report}
\NoBgThispage


\maketitle
\begin{center}
\textbf{INDIAN INSTITUE OF TECHNOLOGY}

kanpur
\end{center}


\vfill
\begin{figure}
\centering
\includegraphics[scale=.1]{IIT_Kanpur_Logo.svg}
\end{figure}
\newpage
\cfoot{\textbf{Keywords:}\textit{Negative Binomial Distribution, Deconvolution and Neural Framework}}
\tableofcontents
\clearpage
\font\myfont=cmr12 at 40pt
\begin{center}
\title{{\myfont Spatial Transcriptomics}}
\end{center}


\section{Abstract}
Spatial Transcriptomics is an overarching term for a range of methods designed for assigning cell types(identified by mRNA readouts) to their locations in the histological sections. This method can also be used to determine subcellular localisation of mRNA molecules.

The Stahl method implies positioning individual tissue samples on the arrays of spatially barcoded reverse transcription primers able to capture mRNA with oligo(dT) tails. Besides oligo(dT) tails and spatial barcode, which indicates the x and y position on the arrayed slide, the probe contains a cleavage site, amplification and sequencing handle, and unique molecular identifier.

In the broader meaning of this term, spatial transcriptomics include methods that can be divided into five principal approaches to resolving spatial distribution of transcripts. They are microdissection techniques, Fluorescent techniques in situ hybridisation methods, in situ sequencing, in situ capture protocols and in silico approaches.

What we do at our labs is in silico analysis. We are provided with the Single-cell data and the spatial transcriptomic data. Single cell data contain the information regarding the distribution of mRNA counts in specific cells. These data are mainly generated by 10X genomics. The spatial transcriptomic data contain RNA distribution at all the spots in the tissue section. 

We run those datasets on the published methods like DestVI, Stereoscope, Seurat, CARD, DSTG, Autogenes and many more. DestVI and Autogenes, and almost all other methods, have two model $sc-model$ and $st- model$. DestVI posits that for each gene $g$ and each cell $n$ , the number of observed transcripts follows a negative binomial distribution.The distribution is parametrized as $(r_{ng},p_{g})$ with mean $\frac{(r_{ng}\* p_{g})}{1-p_{g}}$ and where $p_{g}$ is the gene specific parameter determining the mean-variance relationship at each spot.Parameter $r_{ng}=l_{n}\*\rho_{ng}$ of the negative binomial depends on the type assigned to $c_{n}$, and its overall number of detected molecules $l_{n}$ and a low dimensional latent vector $\gamma_{n}$ which captures the variability within its respective cell types. A neural framework maps $\gamma_{n}$ and $c_{n}$ to $\rho_{ng}$. The other st-model also relies on Negative Binomial distribution. Finally, we deconvolute the sc-model with the st-model getting the final disttribution at each spot.
\vfill





\end{document}